js/tests/unit/tooltip.js

```diff
@@ -680,6 +680,42 @@ $(function () {
     $styles.remove()
   })
 
+  test('should not be adjust position when scrolling', function () {
```

Comment:
I'm surprised I didn't fix this one, my bad!  Now it seems in @cvrebert 's hands, sorry about that!